process/interface.go

```diff
@@ -290,18 +290,23 @@ type MessageHandler interface {
 	IsInterfaceNil() bool
 }
 
-// TopicHandler defines the functionality needed by structs to manage topics and message processors
-type TopicHandler interface {
+type topicHandler interface {
 	HasTopic(name string) bool
 	CreateTopic(name string, createChannelForTopic bool) error
 	RegisterMessageProcessor(topic string, handler p2p.MessageProcessor) error
 }
 
+// TopicHandler defines the functionality needed by structs to manage topics and message processors
+type TopicHandler interface {
```

Comment:
this interface is not used, only the private.

Reply:
It is used in interceptor factories